PatientWebApplication/SearchMicroserviceApi/Model/Equipment.cs

```diff
@@ -0,0 +1,40 @@
+/***********************************************************************
+ * Module:  Equipment.cs
+ * Author:  Lenovo
+ * Purpose: Definition of the Class Manager.Equipment
+ ***********************************************************************/
+
+using System.Collections.Generic;
+
+namespace SearchMicroserviceApi.Model
+{
+    public class Equipment : Entity
+    {
+        public virtual List<ModelRoom> room { get; set; }
+        public string name { get; set; }
+        public int quantity { get; set; }
```

Comment:
Write class properties to start with upper letter and remove "this" in constructors